pkg/build/nodeimage/build_impl.go

```diff
@@ -196,6 +197,18 @@ func (c *buildContext) buildImage(dir string) error {
 	if err := createFile(cmder, containerdConfigPath, containerdConfig); err != nil {
 		return err
 	}
+	containerdRootlessConfig, err := getContainerdConfig(containerdConfigTemplateData{
```

Comment:
we should be able to do this without building a new node-image.
the entrypoint can rewrite this at runtime instead.

Reply:
How can we edit TOML in the entrypoint? Is sed robust enough?

Reply:
it should be, since the entrypoint is tied to the config, and at this point user patches have not yet been applied, so we know what the config looks like.

we can sed on `default_runtime_name =.*` right?